scrapy/extensions/feedexport.py

```diff
@@ -195,8 +195,7 @@ def item_scraped(self, item, spider):
         return item
 
     def _load_components(self, setting_prefix):
-        conf = dict(self.settings['%s_BASE' % setting_prefix])
-        conf.update(self.settings[setting_prefix])
+        conf = dict(self.settings[setting_prefix])
```

Comment:
Also it's probably better to replace that call for `conf = self.settings.getdict(settings_prefix)`.